packages/@react-spectrum/s2/stories/TextField.stories.tsx

```diff
@@ -28,7 +28,11 @@ const meta: Meta<typeof TextField> = {
     label: {control: {type: 'text'}},
     description: {control: {type: 'text'}},
     errorMessage: {control: {type: 'text'}},
-    contextualHelp: {table: {disable: true}}
+    contextualHelp: {table: {disable: true}},
+    placeholder: {control: {type: 'text'}}
```

Comment:
this doesn't get auto added?

Reply:
oh hm, could've swore I needed this but looks fine locally, thanks